client/src/features/project/projectCoreApi.ts

```diff
@@ -109,7 +113,7 @@ function urlWithQueryParams(url: string, queryParams: any) {
 export const projectCoreApi = createApi({
   reducerPath: "projectCore",
   baseQuery: fetchBaseQuery({ baseUrl: "/ui-server/api" }),
-  tagTypes: ["project", "project-status", "ProjectConfig"],
+  tagTypes: ["project-status", "project", "ProjectConfig"],
```

Comment:
I think we should use the same convention as [RTK docs](https://redux-toolkit.js.org/rtk-query/usage/automated-refetching) for tags, which is `CamelCase` like types, interfaces, classes, etc.

```suggestion
  tagTypes: ["ProjectStatus", "Project", "ProjectConfig"],
```

Reply:
I'm not sure about this. We actually use kebab case in all the other tags so far, with the one exception `ProjectConfig` that is pascal case. I personally find pascal case the most confusing choice here since we use that only for types and react components, and I wouldn't consider switching to yet another convention (camel case)

Reply:
The idea of that convention is that the tag type should be the return type of the query (e.g. `builer.query<ProjectStatus, GetProjectStatusParams>({ ... })`). In that way, it makes sense to adopt the same identifier as the type being returned.

Reply:
Also, I am already using that convention in all my RTK endpoints since I followed the docs :wink:

Reply:
I see the reason, but this doesn't seem easy in all cases. E.G: in the KG API we use `KgMetadataResponse`, that doesn't sound like a good tag name.

I'm ok with using pascal case, but since this PR doesn't touch the tag directly I would rather make a PR that changes all the params to a new convention, and add that the coding guides